packages/api/src/models/company.model.ts

```diff
@@ -0,0 +1,26 @@
+import { serial, pgTable, timestamp, text, integer } from "drizzle-orm/pg-core";
+import { users } from "./user.model";
+
+export const companies = pgTable("Companies", {
+  id: serial("id").primaryKey().notNull(),
+  createdAt: timestamp("created_at").defaultNow().notNull(),
+  updatedAt: timestamp("updated_at").defaultNow().notNull(),
+  name: text("name").notNull(),
+  logo: text("logo"),
+  blurb: text("blurb"),
+});
+
+export type CompaniesSchema = typeof companies.$inferSelect;
+
+export const companies_users = pgTable("Companies_users", {
```

Comment:
The table should really be snake_case, and the variable be camelCase.

```suggestion
export const companiesUsers = pgTable("companies_users", {
```